eng/Build.props

```diff
@@ -80,6 +80,9 @@
 
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Build.Framework.15.1.1012.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Build.Utilities.Core.15.1.1012.csproj" />
+
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Bcl.AsyncInterfaces.7.0.0.csproj" />
```

Comment:
You must add all new projects here in the correct dependency order to avoid prebuilts.

Reply:
I thought this was just for projects that must build first? i.e. if you add a new project that no one depends on, it doesn't need to be added here?

This list has nowhere near the full list of projects, correct?

Reply:
> This list has nowhere near the full list of projects, correct?

Correct.  Once the SBRP is included in previous source-built artifacts it can be removed from here.

> I thought this was just for projects that must build first? i.e. if you add a new project that no one depends on, it doesn't need to be added here?

While this is technically true, it really raises havoc when a package with no dependencies is not added and then later in a separate PR a new package which depends on the previous package is added.  Because of this, our policy is to add all new packages here.

Reply:
Okay sounds good.